src/mailadm/conn.py

```diff
@@ -166,17 +145,12 @@ def get_tokeninfo_by_addr(self, addr):
     # user management
     #
 
-    def add_user(self, addr, hash_pw, date, ttl, token_name):
+    def add_user(self, addr, date, ttl, token_name):
         self.execute("PRAGMA foreign_keys=on;")
 
-        token = self.get_tokeninfo_by_name(token_name)
-        if token and token.usecount >= token.maxuse:
-            raise TokenExhausted("token {} is exhausted".format(token_name))
-
-        homedir = self.config.get_vmail_user_dir(addr)
         q = """INSERT INTO users (addr, hash_pw, homedir, date, ttl, token_name)
                VALUES (?, ?, ?, ?, ?, ?)"""
-        self.execute(q, (addr, hash_pw, str(homedir), date, ttl, token_name))
+        self.execute(q, (addr, "mailcow", "mailcow", date, ttl, token_name))
```

Comment:
I didn't want to change the database schema and delete those two columns for backwards compability. OTOH, if anyone uses this branch, they need to migrate to mailcow before anyway.

Reply:
All of this sounds somehow more complicated than just writing a database migration xD I'll just do that now.

Reply:
okaaaay, I see the point. backwards compability is haaard. Yeah, I'll write a README entry